leaf-ui/index.html

```diff
@@ -42,10 +42,11 @@
                 <!-- <div class="btn">Font Size</div> -->
                 <div class="dropdown-toolbar">
                   <a id="btn-clear-all" class='model-clears'>Full Model and Analysis</a>
-                  <a id="btn-clear-analysis">Analysis</a>
                   <a id="btn-clear-elabel" class='model-clears'>Evaluation Labels</a>
                   <a id="btn-clear-flabel" class='model-clears'>Dynamic Function Labels</a>
                   <a id="btn-clear-cycle" class='model-clears'>Cycle Highlighting</a>
+                  <a id="btn-clear-analysis" class='analysis-clears' style="display: none;">Configurations and Results</a>
+                  <a id="btn-clear-results" class='analysis-clears' style="display: none;">Results</a>
```

Comment:
Super minor, but in keeping with the style of the save dropdown would we want to put results before configurations and results? Or maybe just say "Results only"? Idk I may just be weird here but something abt how this reads compared to our other dropdowns just feels slightly off to me. I don't really know a better way though so maybe ignore this and if other people end up feeling similarly or a new phrasing/order comes to us we can potentially make a change then.

Reply:
This is a great point. I've changed it to "Results Only" on top and "Configurations and Results" on the bottom, but I think we eventually need a broader conversation about the style of our dropdowns. In save, each option is "Save ____" while Clear doesn't preface everything with "Clear," and there's also the question of calling them "Analysis Configurations" versus just "Configurations"